msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -133,6 +133,17 @@ To implement messaging extensions, start with [build a messaging extension](../m
 
 The Teams meeting app extensibility allows you to design your app based on participant roles in a meeting.
 
+### Supported capabilities in meeting
+
+| Capability | Support available | In-tenant user type on mobile | Guest user on mobile | Federated or external user on mobile |
+| Scheduled private meetings | ✔️ | | | |
```

Comment:
please remove scheduled here as well. 
Or if we want to be more specific, we can do Scheduled/Recurring private meetings